RxBlocking/README.md

```diff
@@ -18,6 +18,15 @@ extension ObservableType {
 extension ObservableType {
     public func last() throws -> E? {}
 }
+
+extension Observableype {
```

Comment:
I've just realized all of these extensions are on BlockingObservable vs ObservableType :)